src/wolfkrow/core/tasks/task.py

```diff
@@ -365,102 +366,137 @@ def sub_space_for_underscore(string):
 
         return file_path
 
-    def _command_line_sanitize_attribute(self, attribute_name, attribute_value, deadline=False):
-        """ Processes the attribute value to prepare it for use on the command line.
-
-            Does additional processing to prepare the attribute for use on deadline.
+    def _command_line_sanitize_attribute(
+        self, attribute_name, attribute_value, deadline=False
+    ):
         """
+        Processes the attribute value to prepare it for use on the command line.
 
-        # Ensure that these types are wrapped in quotes so they are interpreted 
-        # as a single argument.
-        if (isinstance(attribute_value, dict) or
-            isinstance(attribute_value, list) or
-            isinstance(attribute_value, set)):
-
-            attribute_value = repr(attribute_value)
-
-        # Sometimes the value here will be a unicode or byte string. If that
-        # is the case, then strip the "u" or "b" qualifier at the start of 
-        # the string output of repr.
-        # TODO: We should probably properly handle these special cases properly,
-        # but this seems to work... keep an eye on this.
-        attribute_value = repr(attribute_value)
-        try:
-            if attribute_value.startswith("u'") or attribute_value.startswith("b'"):
-                attribute_value = attribute_value[1:]
-        except Exception:
-            pass
```

Comment:
Ahhh I see why you added the shlex stuff now. You removed this code, so needed an alternative. 

I'm officially calling all other export methods deprecated... So I'm just going to leave this broken... It will all be removed very soon. I am happy to have this problem finally solved.

Reply:
tbh, I don't think this method is required at all anymore.